src/main/java/edu/ucsb/cs156/example/documents/RedditFlair.java

```diff
@@ -0,0 +1,15 @@
+package edu.ucsb.cs156.example.documents;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class RedditFlair {
+    private String e;
+    private String t;
```

Comment:
What are `e` and `t`? Can we use `@JsonProperty` ([docs](https://github.com/FasterXML/jackson-annotations#annotations-for-renaming-properties)) here so that we can give these fields more descriptive names?

Reply:
If I knew what they were, I would.   I honestly don't and would have to go digging for it.

Reply:
I'm also sort of trying to emphasize that the default case is to keep the names of the Java attributes matching the names of the JSON keys.    I see your point, but I wonder if that doesn't muddy the waters in this case?